### DISCUSSION TITLE
Can’t follow users at other instances

### DISCUSSION THREAD

**Discussion by Hudsoneric** [mastodon/mastodon#24561] [Q&A]
New user. Can find users at other instances, can follow some, but some don’t show up in my app or web when I search for them. 

Is there a way to figure out what is wrong?

**Comment by yelvington** [comment]
The search will find only those users who are "known" to the Mastodon instance on which your account is located, unless you enter a fully qualified name (@username@example.com), which will cause your Mastodon instance to try to go check the named site.

**Comment by Hudsoneric** [comment]
Thanks for your reply.

I am using the fully qualified name but on both my web and app versions of Mastodon, search can’t find him.

It’s hard to know, in the early days, when things seemingly go awry, whether something is broken or actually working exactly as intended.

**Comment by daSmirnov** [comment]
If its working for some accounts but not others it could be those servers or users have been blocked.  Is this on someone else's server or your own?

**Comment by williewillus** [comment]
Bumping this. I'm trying to follow an account on another server, the two servers are small/niche but have not blocked each other, and neither the full http url or double-at notation work to find this user, and they cannot find me in return. The failure mode is very non-obvious as you just see "no results".

Can it be surfaced better?
